.github/workflows/release-secure-partitions.yml

```diff
@@ -54,7 +54,7 @@ jobs:
   publish:
     name: Publish Rust Partitions
     needs: build
-    if: github.event_name == 'release'
+    # if: github.event_name == 'release'
     runs-on: ubuntu-latest
 
     permissions:

```

Comment:
What is the point of this change? it seems to drop the release check but still does a release upload for a fixed version, why?

Reply:
Yeah, I just want this to run and make sure everything can go through fully. Will revert this change back to its original form if that tests works out.